Packs/RecordedFuture/Integrations/RecordedFuture/RecordedFuture.py

```diff
@@ -1357,6 +1371,122 @@ def __build_links_context(self, links_data: Dict[str, List], markdown: str) -> C
                     outputs_key_field=""
                 )
 
+    def fetch_incidents(self, rule_names_arg: Optional[str], fetch_time: str) -> None:
+        if rule_names_arg:
+            rule_names = rule_names_arg.split(';')
```

Comment:
Why split by `;` ?

Reply:
That is because we specify that in the description for rule names, and that is the format we want to get.
```
-display: Rule names to fetch alerts by, separated by semicolon. If empty, all alerts
    will be fetched
 ```